### DISCUSSION TITLE
Move system on a larger card

### DISCUSSION THREAD

**Discussion by grychofr** [mcguirepr89/BirdNET-Pi#1117] [Q&A]
Hello,

I have several stations running in France with PI4 and 64GB SD card. My first station is on a pi3 with a 8 gb card (Saint Paul Trois Chateaux), It works very well during 4 months. And stops on 25th january : no more on birdweather, and I cannot connect locally (either with Wifi and cable, with cable it connects to the router for a very short time). I guess the card is full. On a Windows PC, with a disk manager I can see that the card is full. 

This is my firs station, and I do not remember if I choose Keep or purge files in advanced seting. I can read "When the disk becomes full, you can choose to 'purge' old files to make room for new ones or 'keep' your data and stop all services instead.". It seems to explain why it does not work anymore. All services are stopped.

I think the best is to make a new install on a larger card. But what is the better way to recover database ? And I guess if I enter the same birdweather ID, it will be considered as the same stations by Birdweather.

Thanks

**Comment by Svardsten53** [comment]
You can take an image copy of the old card and restore it to the new card. Depending of what tool you are using for this you might have to expand the file system on the new card. 

`sudo raspi-config`
Select 6. Advanced options
Select A1 Expand Filesystem
